### DISCUSSION TITLE
Using bundle() with cloud-hosted apps

### DISCUSSION THREAD

**Discussion by uluhonolulu** [remotion-dev/remotion#1511] [Q&A]
When you publish your React app to the cloud (e.g. Azure static), the deployment script would build the app and copy the contents of the build folder to the website's location. Therefore, we cannot run the bundler as a part of a request handler -- the source code is not where the app would expect it.

Three things I can think of here:
1. Create the bundle on the dev box and commit it to the repository before pushing.
2. Create it as a part of the build process.
3. Create it using a GitHub action.

Any recommendations here?

**Comment by JonnyBurger** [comment]
Discussion was continued here: https://github.com/remotion-dev/remotion/issues/1513
